docs/src/app/pages/codibotintro.md

```diff
@@ -0,0 +1,35 @@
+# Platform: MIT App Inventor Codi Bot
+
+The MIT App Inventor Codi Bot is a hands-on IoT kit. Our adorable mascot can be controlled via App Inventor through Bluetooth communication. This educational kit is easy to use and guides users through the whole process of development, from building a robot to programming it. 
+
+<div style="text-align: center; font-size: 75%; margin: 16pt 0;">
+![Image of the MIT App Inventor Codi Bot.](/assets/howtos/codibot1.jpg)
+
+</div>
+
+## Parts list (<a href="https://www.robotkingdom.com.tw/product/mit-app-inventor-codi-bot/" target="_blank">Purchase link)</a>:
+* <a href="https://www.robotkingdom.com.tw/product/linkit-7697/" target="_blank">LinkIt 7697 MCU board</a>, 1
+* <a href="https://www.robotkingdom.com.tw/product/linkit-7697%E6%A9%9F%E5%99%A8%E4%BA%BA%E6%93%B4%E5%85%85%E6%9D%BF-robot-shield-for-linkit-7697/" target="_blank">Robot Shield for LinkIt 7697 V2.0</a>, 1 (with an onboard buzzer) 
+* RGB LED (common cathode), 1
+* LED stripe green (left wing), 1
+* LED stripe purple (right wing), 1
+* Ultrasonic sensor (HC-SR04), 1
+* mini servo (SG90), 2
+* female-female jump wires, 10
+* screws (M3x8, 7pcs; M2x12, 2pcs; M2x6, 12pcs)
+* Acrylic pieces (base, wings), 2 pieces.
+* Cow leather pieces (stripes and body cusions), 4 pieces.
+* 3D printed parts, 9 pieces.
+
+## Please follow our tutorials to get started:
+
+Compared with [LinkIt 7697 tutorials](#/linkit/linkit7697), tutorials for the MIT App Inventor Codi Bot use an extension of the LinkIt 7697 dev board instead of low-level Bluetooth commands.
+
+<ul>
+<li><a href='/assets/howtos/MIT_App_Inventor_CodiBot_UserManual.pdf' target='_blank'>User manual</a> &mdash; Follow this manual to assemble the MIT App Inventor Codi Bot.</li>
```

Comment:
Has Hedge reviewed the PDFs for spelling and grammar?

Reply:
Also, is there a Google Doc somewhere, or something similar? I have some comments on the PDFs as well that I'd like to share.

Reply:
no problem. mail to you